platform/packages/currency/src/test/group.rs

```diff
@@ -25,6 +25,19 @@ impl Group for SuperGroup {
     const DESCR: &'static str = "super_group";
     type TopG = Self;
 
+    fn currencies() -> impl Iterator<Item = CurrencyDTO<Self>> {
+        [
+            SuperGroupTestC1::definition().dto(),
+            SuperGroupTestC2::definition().dto(),
+            SuperGroupTestC3::definition().dto(),
+            SuperGroupTestC4::definition().dto(),
+            SuperGroupTestC5::definition().dto(),
+        ]
+        .into_iter()
+        .map(ToOwned::to_owned)
```

Comment:
Please refer to previous comment.
```suggestion
        .copied()
```